test/readme.jl

```diff
@@ -2,7 +2,8 @@ using Weave
 using Documenter
 
 function buildREADME()
-    weave("../README.jmd", doctype= "github", throw_errors=true, cache=:refresh, args=Dict(:seed => 6))
+    set_chunk_defaults!(:error=>false)
```

Comment:
See
http://weavejl.mpastell.com/stable/chunk_options/#Evaluation

The `false` feels backwards to me, but nevertheless it now should stop if errors are thrown so the readme functions as a test.